defaults/main.yml

```diff
@@ -1,30 +1,31 @@
 ---
 # defaults file for ansible-role-openio-keepalived
 
-keepalived_conf_template: keepalived.conf.j2
 keepalived_conf_path: /etc/keepalived/keepalived.conf
 
 keepalived_global_defs_notification_email: support@openio.io
 keepalived_global_defs_notification_email_from: "{{ inventory_hostname }}@openio.io"
 keepalived_global_defs_smtp_server: 127.0.0.1
 keepalived_global_defs_smtp_connect_timeout: 30
+keepalived_track_script: check_haproxy
+keepalived_auth_type: PASS
 
 keepalived_vrrp_instances:
   VI_01:
-    state: MASTER
-    interface: "{{ ansible_default_ipv4.interface }}"
-    virtual_router_id: '51'
-    priority: '100'
-    advert_int: '1'
+    state: "{{ instance_state|default('BACKUP') }}"
+    interface: "{{ instance_interface }}"
+    virtual_router_id: "{{ instance_vrid|default('51') }}"
+    priority: "{{ instance_priority|default('100') }}"
+    advert_int: "{{ instance_advert_int|default('1') }}"
     authentication:
-      auth_type: PASS
-      auth_pass: AUTH_PASS
+      auth_type: "{{ keepalived_auth_type }}"
+      auth_pass: "{{ instance_pwd }}"
```

Comment:
Should default to AUTH_PASS here, even if unsecure by default.
I would prefer `instance_password` instead of `instance_pwd`, it's easier to grep password you need to change ;-)